src/builders/GroupStep.ts

```diff
@@ -57,7 +57,8 @@ export class GroupStep implements StepBuilder, KeyBuilder, LabelBuilder, Depende
 
   build(): GroupStepSchema {
     const step: GroupStepSchema = {
-      group: null,
+      // Workaround until the schema is updated to make `group` nullable
+      group: this.#labelHelper.build().label ?? '',
```

Comment:
Workaround for now, will create a PR to make this key nullable in the jsonSchema